Sia/Buffers/ArrayBuffer.cs

```diff
@@ -46,17 +54,16 @@ private static int CalculateArraySize(int requiredCapacity)
     public readonly bool IsAllocated(int index)
         => index < _values.Length;
 
-    [MethodImpl(MethodImplOptions.AggressiveInlining)]
-    public readonly ref T GetRef(int index)
```

Comment:
The core of the GetRef is to throw error if need. But we need keep data safe in production